python/lsst/sims/maf/batches/metadataBatch.py

```diff
@@ -130,7 +144,7 @@ def metadataBasics(value, colmap=None, runName='opsim',
     return mb.makeBundlesDictFromList(bundleList)
 
 
-def allMetadata(colmap=None, runName='opsim', sqlconstraint='', metadata='All props'):
+def allMetadata(colmap=None, runName='opsim', sqlConstraint='', metadata='All props'):
```

Comment:
Should single visits be two words in the docstring?

Reply:
Done.